apps/frontend/assets/src/omg-page-api-detail-accounts/index.js

```diff
@@ -0,0 +1,356 @@
+        )
+      case 'account.id':
+        return (
+          <Id maxChar={20}>
+            {accountId || '-'}
+          </Id>
+        )
+      case 'account.parent_id':
+        return _.get(rows, 'account.parent_id', '-')
+      case 'created_at':
+        return moment(data).format()
+      case 'role':
+        const options = [
+          { key: 'admin', value: 'Admin' },
+          { key: 'viewer', value: 'Viewer' }
+        ]
+        return (
+          <Select
+            onSelectItem={(item) => handleRoleSelect(data, item.key, accountId)}
+            style={{ width: '150px' }}
+            noBorder
+            value={_.startCase(data)}
+            options={options.filter(i => i.key !== data)}
+          />
+        )
+      default:
+        return data
+    }
+  }
+
+  const getRows = () => {
+    return memberships && memberships.length
+      ? memberships.map(membership => ({
+        id: membership.account_id,
+        ...membership
+      }))
+      : []
+  }
+
+  return (
+    <>
+      <AssignAccountToKeyModal
+        keyId={keyId}
+        open={assignAccountToKeyModal}
+        onRequestClose={() => setAssignAccountToKeyModal(false)}
+        onSubmitSuccess={refetch}
+      />
+
+      <Modal
+        shouldReturnFocusAfterClose={false}
+        isOpen={roleModalOpen}
+        onRequestClose={closeModals}
+        contentLabel='assign-role-modal'
+        shouldCloseOnOverlayClick
+      >
+        {renderAssignRoleModal()}
+      </Modal>
+
+      <Modal
+        shouldReturnFocusAfterClose={false}
+        isOpen={deleteModalOpen}
+        onRequestClose={closeModals}
+        contentLabel='delete-role-modal'
+        shouldCloseOnOverlayClick
+      >
+        {renderDeleteRoleModal()}
+      </Modal>
+
+      <BreadContainer>
+        <Breadcrumb
+          items={[
+            <Link key='keys-main' to={`/keys/${keyType}`}>Keys</Link>,
+            <Link key='key-detail' to={`/keys/${keyType}/${keyId}`}>
+              <Id withCopy={false} maxChar={20} style={{ marginRight: '0px' }}>{keyId}</Id>
+            </Link>,
+            <span key='assigned-accounts'>Assigned Accounts</span>
+          ]}
+        />
+      </BreadContainer>
+      <TopNavigation
+        title={
+          <TitleContainer>
+            <Icon name='Key' />
+            <span>Assigned Accounts</span>
+          </TitleContainer>
+        }
+        buttons={[
+          <Button
+            key='assign-new-account-button'
+            styleType='secondary'
+            size='small'
+            onClick={() => setAssignAccountToKeyModal(true)}
+          >
+            <Icon name='Plus' style={{ marginRight: '10px' }} />
+            <span>Assign New Account</span>
+          </Button>
+        ]}
+        normalPlaceholder='Search by Account ID'
+        divider={false}
+      />
+      <SortableTable
+        rows={getRows()}
+        rowRenderer={rowRenderer}
+        columns={columns}
+        loadingStatus={membershipsLoading}
```

Comment:
Can you double check the `membershipsLoading` ? SortableTable is giving weird loading animation ( blinking ) 😆